Tasks/FuncToolsInstallerV0/task.json

```diff
@@ -0,0 +1,50 @@
+{
+    "id": "A22A6EB4-FE21-442E-ABCE-5B2B09CFECE3",
+    "name": "FuncToolsInstaller",
+    "friendlyName": "Install Azure Func Core Tools",
+    "description": "Install Azure Func Core Tools",
+    "helpUrl": "https://docs.microsoft.com/azure/devops/pipelines/tasks/tool/func-tools-installer",
+    "helpMarkDown": "[Learn more about this task](https://docs.microsoft.com/azure/devops/pipelines/tasks/tool/func-tools-installer)",
+    "category": "Tool",
+    "visibility": [
+        "Build",
+        "Release"
+    ],
+    "author": "Microsoft Corporation",
+    "version": {
+        "Major": 0,
+        "Minor": 159,
+        "Patch": 0
+    },
+    "preview": true,
+    "demands": [],
+    "satisfies": [
```

Comment:
What does this do? 
I couldn't find this reference in the task.json schema.

Reply:
Its a way of fulfilling a demand. A task can `demand` `Func` so that it doesn't start if this installer task is not there. Following the pattern in other Installer tasks.